pkg/systemd/host.js

```diff
@@ -566,6 +576,20 @@ PageServer.prototype = {
             refresh_os_updates_state();
         });
 
+        function refresh_service_failures() {
+            const status = page_status.get("system/services");
+
+            $("#system_information_service_failures").empty();
```

Comment:
I feel a bit uneasy about adding new jQuery stuff. Would it be much more effort to use standard JS API for this code?

Reply:
I think the way forward for this page is to rewrite it in React, not with the plain JS API, no?  As such, this would feel like a step in the wrong direction.

Reply:
Agreed. Resolving the jquery comments.